wordstats/edit_distance.py

```diff
@@ -0,0 +1,32 @@
+import codecs
+import configparser
+from .cognate_files_path import *
+from .edit_distance_function_factory import WordDistanceFactory
+
+
+class LanguageAwareEditDistance(WordDistanceFactory):
+    def __init__(self, primary_language, secondary_language):
```

Comment:
short comment on what is the meaning of this class would be good. 
also, i see no language awareness in the class. why?